### PR TITLE
Response tracker & survey score APIs modified

### PR DESCRIPTION
**Pull request by Ajay-Maury**

### CODE REVIEW

File: src/response-tracker/response-tracker.controller.ts [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+  Param,
   ParseIntPipe,
   ParseUUIDPipe,
+  Patch,
+  Post,
+  Res,
 } from "@nestjs/common";
-import { ResponseTrackerService } from "./response-tracker.service";
 import { ApiOperation, ApiResponse, ApiTags } from "@nestjs/swagger";
+import { getPrismaErrorStatusAndMessage } from "src/utils/utils";
```
Comment (prashantesmagico) [inline src/response-tracker/response-tracker.controller.ts:16]: import properly

File: src/survey-score/survey-score.service.ts [hunk 2]
```diff
@@ -4,13 +4,13 @@ import {
   NotFoundException,
 } from "@nestjs/common";
 import { ResponseTrackerStatusEnum, SurveyStatusEnum } from "@prisma/client";
```
Comment (prashantesmagico) [inline src/survey-score/survey-score.service.ts:6]: import properly it causes issue in test cases

**Review by prashantesmagico — CHANGES_REQUESTED** [review]

File: src/response-tracker/response-tracker.controller.ts (hunk 1 above)
Comment (Ajay-Maury) [reply src/response-tracker/response-tracker.controller.ts:16]: Test cases are in another PR changing here may cause merge conflicts

File: src/survey-score/survey-score.service.ts (hunk 2 above)
Comment (Ajay-Maury) [reply src/survey-score/survey-score.service.ts:6]: Test cases are in another PR changing here may cause merge conflicts

**Review by VamshiBatta07 — APPROVED** [review]

**Review by prashantesmagico — APPROVED** [review]

**Pull request merged**